src/Target.cpp

```diff
@@ -661,6 +661,10 @@ bool Target::supported() const {
     return !bad;
 }
 
+bool Target::defined() const {
```

Comment:
defined is very overloaded. Other ideas: fully_specified, has_unknown_fields, has_unknowns, all_fields_set

Reply:
I vote for `has_unknowns()` of those.